src/content/docs/apis/intro-apis/introduction-new-relic-apis.mdx

```diff
@@ -72,20 +72,77 @@ This document provides examples and reference information for our API endpoints.
   To use APIs and SDKs, as well as the rest of our [observability platform](https://one.newrelic.com), join the New Relic family! [Sign up](https://newrelic.com/signup) to create your free account in only a few seconds. Then ingest up to 100GB of data for free each month. Forever.
 </Callout>
 
+
+## Ingest APIs for core data types [#data-type-apis]
+
+We have [many solutions for reporting data](/docs/telemetry-data-platform/get-started/introduction-new-relic-data-ingest-apis-sdks), and our APIs are one of these ways. These APIs can be used directly, but they're also the underlying ingest route for many of our more curated tools, such as our [open source telemetry integrations](/docs/integrations/open-source-telemetry-integrations/get-started/introduction-new-relics-open-source-telemetry-integrations) and our [Telemetry SDKs](/docs/telemetry-data-platform/ingest-apis/telemetry-sdks-report-custom-telemetry-data). 
```

Comment:
That first sentence feels a little too casual to me...

How about something more like: "Our APIs are only one of many solutions you can use to report data to New Relic."